metadata/utils.go

```diff
@@ -43,6 +43,9 @@ const (
 
 	// MaxBulkAwaitDurationKey is the key for the max bulk await duration in the metadata.
 	MaxBulkAwaitDurationMilliSecondsKey string = "maxBulkAwaitDurationMilliSeconds"
+
+	// MaxBulkPubBytes defines the maximum bytes to publish in a bulk publish request metadata.
+	MaxBulkPubBytes string = "maxBulkPubBytes"
```

Comment:
Do you think this could make sense in _any_ publish request and not just bulk? For example, I see that you did it because ASB has a limit of 16MB.... Perhaps it makes sense to use this as a hard limit in _all_ publish requests?